### ISSUE TITLE
Unable to display markers

### ISSUE THREAD

**Issue by Bog97** [zcreativelabs/react-simple-maps#92]
Hey zcreativelabs, thanks for creating react-simple-maps. 

I am pretty new to react and front end development so I apologize if I am making some sort of dumb mistake. I have went through my code and compared it to multiple examples that you have up on your page and for the most part everything looks the same/ correct. I've been struggling for a while now to understand why adding markers to my map is breaking my application. Without including Markers and Marker my application works fine, but as soon as I try to add one marker my application no longer renders.

This is all of my code relating to the map: 

```javascript
import React, { Component } from "react";
import { ComposableMap, Geographies, Geography, Markers, Marker, ZoomableGroup, Lines, Line } from "react-simple-maps";
import classes from "./Map.css";
import ReactTooltip from "react-tooltip";


const markers = [
    { markerOffset: -25, name: "Polaris", coordinates: [47.4235, 120.3103] }, //47.4235, 120.3103
    { markerOffset: -25, name: "Titan", coordinates: [32.7767, 96.7970] }, //32.7767, 96.7970
];

class Map extends Component {
    componentDidMount() {
        setTimeout(() => {
            ReactTooltip.rebuild()
        }, 100)
    }

    render() {
        return (
            <div className={classes.Map} >
                <ComposableMap
                    className={classes.Map}
                    width={1000}
                    height={450}
                    projection="albersUsa"
                    projectionConfig={{ scale: 800 }}>
                    <ZoomableGroup disablePanning>
                    <Geographies
                        disableOptimization
                        geography="https://raw.githubusercontent.com/zcreativelabs/react-simple-maps/master/examples/albers-usa/static/states.json" >
                        {(geos, proj) =>
                            geos.map((geo, i) => (
                                <Geography
                                    className={classes.Map}
                                    key={i}
                                    cacheId={geo.properties.ID_1}
                                    geography={geo}
                                    projection={proj}
                                    style={{
                                        default: {
                                            fill: "black",
                                            stroke: "#f73aa9",
                                            strokeWidth: 0.3,
                                            outline: "none",
                                        },
                                        hover: {
                                            fill: "#ec008a",
                                            stroke: "white",
                                            strokeWidth: 1,
                                            outline: "none",
                                        },
                                        pressed: {
                                            fill: "#ef32a1",
                                            stroke: "black",
                                            strokeWidth: 1.2,
                                            outline: "none",
                                        }
                                    }}
                                    data-tip={geo.properties.NAME_1}
                                    onClick={() => {
                                        console.log('This state was clicked:', geo.properties.NAME_1,
                                            "State's Properties:", geo)
                                    }}
                                />
                            ))
                        }
                    </Geographies>
                    {/* <Lines>
                        <Line
                            line={{
                                coordinates: {
                                    start: [0, 0],
                                    end: [-99.1, 19.4]
                                }
                            }}
                        />
                    </Lines> */}
                    <Markers>
                            {markers.map((marker, i) => (
                                <Marker
                                    key={i}
                                    marker={marker}
                                    style={{
                                        default: { fill: "#FF5722" },
                                        hover: { fill: "#FFFFFF" },
                                        pressed: { fill: "#FF5722" },
                                    }}
                                >
                                    <circle
                                        cx={0}
                                        cy={0}
                                        r={10}
                                        style={{
                                            stroke: "#FF5722",
                                            strokeWidth: 3,
                                            opacity: 0.9,
                                        }}
                                    />
                                    <text
                                        textAnchor="middle"
                                        y={marker.markerOffset}
                                        style={{
                                            fontFamily: "Roboto, sans-serif",
                                            fill: "#607D8B",
                                        }}
                                    >
                                        {marker.name}
                                    </text>
                                </Marker>
                            ))}
                        </Markers> 
                     </ZoomableGroup> 
                </ComposableMap>
                <ReactTooltip />
            </div>
        )
    }
}

export default Map;
```

When I try to use markers without using ZoomableGroup I get these errors: 
<img width="957" alt="error2" src="https://user-images.githubusercontent.com/32381563/44046783-6a6d3de4-9ee1-11e8-8695-53f93405d1d3.PNG">

When I do include ZoomableGroup I get these errors:
<img width="954" alt="error1" src="https://user-images.githubusercontent.com/32381563/44046815-7e804d94-9ee1-11e8-807a-ec4642c686e2.PNG">

Is using the ZoomableGroup object required for markers? Either way why am I getting these errors?
Any advice or help of any kind would be awesome and thanks again!

**Comment by zimrick** [comment]
Hi @Bog97,

From what I see there could be a problem with your coordinates. The albersUsa projection does not cover the entire globe, so any coordinates outside of the scope of this projection will cause problems. Note that the order of coordinates in react-simple-maps is `[lng, lat]`, it seems your coordinates are specified as `[lat, lng]` so you might get strange results from that as well.

I copied your code over to codesandbox and switched the coordinates around. I got this:

<img width="1013" alt="screen shot 2018-08-14 at 14 58 51" src="https://user-images.githubusercontent.com/7160099/44093061-a1f5eecc-9fd2-11e8-9b9d-a5ae0e076176.png">

Hope this is helpful to you.
